content/en/community/hackathons/usoc22/contributing-to-unikraft/index.md

```diff
@@ -32,6 +32,10 @@ weigth: 50
 
 {{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Summary.md" markdown="true" >}}
 
+## Quiz
+
+If you reached this far, please do a 5 minutes tour through [this quiz](https://docs.google.com/forms/d/e/1FAIpQLSdC-EYerUXXZ_8llf52tIftw2Ku7UT3LQzssoq91sVm0RpHnA/viewform?usp=sf_link) to assess your current understanding of the session.
```

Comment:
The quiz should be after the practical work, and before the feedback form.

Also, maybe you can generate that shorter link to the form and change it here.